### PR TITLE
feat(dataman):add dataman to the vvplus repo

### PR DESCRIPTION
**Pull request by chandantiwari1**
Add dataman to the vvplus app and setup the github actions ci-cd pipelin on build same as meway-tablet. fix monorepo issues and configure eslint config custom for the project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -104,10 +104,3 @@ dist
 .tern-port
 
 **/build/
-
-# Do not commit lock files, they tend to go out of sync
```
Comment (sutyum) [inline .gitignore:108]: why did you delete these? they are still useful

File: apps/Dataman/.expo/devices.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "devices": []
```
Comment (sutyum) [inline apps/Dataman/.expo/devices.json:2]: what is this for?

File: apps/designer/src/components/FormComponents/SaveNumberForm.tsx [hunk 3]
```diff
@@ -2,10 +2,13 @@ import React from "react";
 import { boolean } from "zod";
```
Comment (sutyum) [inline apps/designer/src/components/FormComponents/SaveNumberForm.tsx:2]: this file should not be a part of this PR

**Review by sutyum — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (chandantiwari1) [reply .gitignore:108]: no such type of file was in our directory

File: apps/designer/src/components/FormComponents/SaveNumberForm.tsx (hunk 3 above)
Comment (chandantiwari1) [reply apps/designer/src/components/FormComponents/SaveNumberForm.tsx:2]: Ok sir I will delete this file

File: apps/Dataman/.expo/devices.json (hunk 2 above)
Comment (chandantiwari1) [reply apps/Dataman/.expo/devices.json:2]: in this devices array we can define the device type and their configuration. may be useful in the future

File: .gitignore (hunk 1 above)
Comment (sutyum) [reply .gitignore:108]: it is still useful to keep .env as many users might end up creating their own and should not accidentally commit it

**Review by sutyum — APPROVED** [review]
